pkg/cni/plugin_test.go

```diff
@@ -0,0 +1,195 @@
+package cni_test
+
+import (
+	"fmt"
+
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containernetworking/plugins/pkg/testutils"
+	"github.com/kubevirt/macvtap-cni/pkg/cni"
+	"github.com/kubevirt/macvtap-cni/pkg/util"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vishvananda/netlink"
+)
+
+const LOWER_DEVICE = "eth0"
+
+var _ = Describe("Macvtap device plugin", func() {
+	var originalNS ns.NetNS
```

Comment:
You should add a Context into the Describe. The Context describes what is available for It, e.g. `Context("with configuration set"). This context is achieved by `BeforeEach`. `Describe` is just for the general description of the suite

Reply:
Done.